src/themes/teams/components/Tree/treeTitleStyles.ts

```diff
@@ -0,0 +1,11 @@
+import { ICSSInJSStyle } from '../../../types'
+
+const treeTitleStyles = {
+  root: ({ variables }): ICSSInJSStyle => ({
+    padding: '.1rem 0',
```

Comment:
I'm not sure, but may be we should use there `pxToRem()` util?

/cc @miroslavstastny @kuzhelov

Reply:
yes, absolutely valid point - @priyankar205, please, consider to use `pxToRem` utility for all sizes that should scale with page's font size being changed, similar how it is done for button styles: https://github.com/stardust-ui/react/blob/master/src/themes/teams/components/Button/buttonStyles.ts#L83